test/.eslintrc.yml

```diff
@@ -10,3 +10,6 @@ settings:
 globals:
   document: false
   Element: false
+rules:
+  no-var: 2
+  prefer-const: 2
```

Comment:
Any plan to migrate the source code off var and onto let/const as well? I noticed there's no build step I imagine it's about browser compatibility?

Reply:
> I noticed there's no build step I imagine it's about browser compatibility?

Yep! It's that! So, definitely worth ticketing / reviewing in the future if browser matrix is ready to the point to do that!